src/fmod_studio.rs

```diff
@@ -16,15 +16,24 @@ pub struct FmodStudio(pub Studio);
 impl FmodStudio {
     pub(crate) fn new(banks_paths: &[&'static str]) -> Self {
         let studio = Self::init_studio();
-        let project_root = var("CARGO_MANIFEST_DIR").unwrap();
 
         banks_paths.iter().for_each(|bank_path| {
-            let mut path = canonicalize(Path::new(bank_path))
-                .expect("Failed to canonicalize provided audio banks directory path.");
+            println!("{} tes test", bank_path);
+            let mut path: PathBuf = bank_path.into();
 
-            trace!("Canonicalized audio banks directory path: {:?}", path);
+            trace!("audio banks directory path: {:?}", path);
 
             if path.is_relative() {
+                let project_root: PathBuf = match var("CARGO_MANIFEST_DIR") {
+                    Ok(path) => path.into(),
+                    Err(_) => std::env::current_exe()
+                        .expect("unable to resolve executed binary path")
+                        .parent()
+                        .expect("binary must be in a resolvable directory")
+                        .to_path_buf(),
+                    // Err(_) => std::env::current_dir().unwrap(),
```

Comment:
Something is commented out.

Reply:
I kept the cwd/current_dir alternative around for a bit as a fallback option, but ultimately going by binary location seems more sensical to me.